networkConfig.js

```diff
@@ -28,13 +19,13 @@ export default {
     networkName: 'Ethereum Mainnet',
     deployedBlock: 9116966,
     rpcUrls: {
-      Infura: {
-        name: 'Infura',
-        url: `https://mainnet.infura.io/v3/${INFURA_KEY}`
+      Securerpc: {
+        name: 'Securerpc',
+        url: `https://api.securerpc.com/v1`
       },
-      Alchemy: {
-        name: 'Alchemy',
-        url: `https://eth-mainnet.alchemyapi.io/v2/${ALCHEMY_MAINNET_KEY}`
+      Home_Hosted: {
+        name: 'Your own non‑censoring rpc.<br>Get a snapshot from http://archivenode.net',
```

Comment:
```suggestion
        name: 'Your own non‑censoring rpc.'
```
I don't think arrhdve.dowload is an appropriate place to link out to here.  It is very confusing for nor Al users, it only lasts a couple different clients, it links to archive nodes which is unnecessarily large, it will take longer for a user to sync than snap syncing with a default client, it isn't https, and it isn't a trusted third party within the community like the official client release pages.

Reply:
@ytrezq I will list Erigon on http://archivenode.net before that repo is minified or deployed again. Though TheGraph censored us. So, we need `parity_tracecallmany`.

Download is done over Torrent or Edondey anyway.

Reply:
As discussed in Matrix, there is no need for traceMany.  You only need a regular Ethereum client.  CloudFl're's public gateway will work for example.

Reply:
@MicahZoltu cloudflare contains data only for the last block. It doesn’t even support eth_filter.

Reply:
you dont nued eth_filter, you need eth_getLogs.

Reply:
Though, the UI may *currently* use filter (I haven't looked), but it isn't required.

Reply:
`eth_getLogs` requires `eth_filter`

Reply:
No it doesn't.

Reply:
From documentation, https://openethereum.github.io/JSONRPC-eth-module#eth_getlogs requires an `eth_filter` object created earlier.

Reply:
And for Erigon, the issue is https://github.com/ledgerwatch/erigon/issues/4153#issuecomment-1240123160

Reply:
> From documentation, https://openethereum.github.io/JSONRPC-eth-module#eth_getlogs requires an `eth_filter` object created earlier.

That is just saying that the shape of the return value is the same as the shape of the return value of `eth_getFilterChanges`.  There is no dependency between the two.

I believe all clients and hosted nodes implement `eth_getLogs`, but a subset implement/offer `eth_getFilterChanges` because it requires server side state.

Reply:
Ok, then you never read web3 demonstrating how to use the enpoint.

Reply:
The filter stuff is for receiving "push" notifications of new logs as they come in.  Over http it still requires polling.  You can implement the entire workflow client side without dependence on server state, which I recommend anyway.  Broadly speaking, I recommend against using eth_getFilterChanges even if it is available to you because the model for it is broken given that reorging and disconnects can happen.  Plus it doesn't really work at all for distributed backends like cloudflare and Infura.

Reply:
It’s required for past logs too. I used to modify https://github.com/blockchain-etl/ethereum-etl and this is how it’s done. Turned out I’m needing a full node. This is also how I learned Nethermind is returning garbage in the case of Gnosis even when using ipc or websockets.

Reply:
I have written a *lot* of software/scripts that uses `eth_getLogs` without filter, and I have advised multiple projects to stop using filters.  I'm quite confident you do not need filters or tracing to accomplish the needs of Tornado Classic.

That being said, I don't see parity_tracecallmany, eth_getFilterChanges, or eth_getLogs used in this repository at all, so at the moment this discussion is somewhat tangential to this changes on so we should move it elsewhere and focus on this change.